Assets/Scripts/HandController.cs

```diff
@@ -45,14 +47,17 @@ void Start()
     }
 
     private void buttonCallBack(Button buttonPressed) {
-        bool isOut = false;
+        //bool isOut = false;
+        deliveries += 1;
         int newScore = getRuns(buttonPressed, game.getIsPlayerPlaying());
         if(newScore == -1){
-            isOut = true;
+            //isOut = true;
+            wickets += 1;
         }
         bool playerIsPlaying = game.getIsPlayerPlaying();
         bool isFirstInnings = game.getIsFirstInnings();
-        if (isOut){
+        //if (isOut){
+        if(wickets == 10 || deliveries == 30){

```

Comment:
1. Less than or equal to both
2. Make wickets and overs selectable in the previous screen